### PR TITLE
feat: check secrets in js files

### PR DESCRIPTION
**Pull request by Mandeep08punia**

### CODE REVIEW

File: test/unit/unit-test1.spec.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
+
```
Comment (abose) [inline test/unit/unit-test1.spec.js:1]: Change file name to something more meaningful. Eg. `secres-scnner.spec.js` or something more appropriate.

**Review by abose — CHANGES_REQUESTED** [review]

File: test/unit/unit-test1.spec.js (hunk 1 above)
Comment (Mandeep08punia) [reply test/unit/unit-test1.spec.js]: Changed the file name to secrets-scanner.spec.js

File: package.json [hunk 2]
```diff
@@ -39,14 +39,17 @@
...
+    "@commitlint/cli": "^16.2.1",
+    "@commitlint/config-conventional": "^16.2.1",
     "c8": "7.11.0",
     "chai": "4.3.6",
     "eslint": "8.9.0",
     "husky": "7.0.4",
     "mocha": "9.2.1"
   },
   "dependencies": {
+    "fs": "0.0.1-security",
```
[the remark this answers is not in the document]
Comment (Mandeep08punia) [reply package.json]: Removed the dependency

**Review by abose — APPROVED** [review]

**Comment by abose** [comment]
LGTM Merging.
Thanks @Mandeep08punia

**Pull request merged**